src/Uno.UI/UI/Xaml/Controls/Flyout/FlyoutBase.cs

```diff
@@ -20,7 +21,7 @@
 
 namespace Windows.UI.Xaml.Controls.Primitives
 {
-	public partial class FlyoutBase : DependencyObject
+	public abstract partial class FlyoutBase : DependencyObject
```

Comment:
This should be reverted, it's not `abstract` on UWP

Reply:
The constructor is protected, so it's the same...